fittable.R

```diff
@@ -0,0 +1,36 @@
+fittable <- function(..., indices="robust", all=F) {

```

Comment:
Nebude lepsi do funkce rovnou passovat list modelu? Nejak takto:

` fittable <- function(model_list, indices = 'robust', all = FALSE){
if (!class(model_list) == 'list'){error('model_list' needs to be a list)}
.
.
}

fittable(model_list = list(mod1, mod2, mod3)), indices, all)`

mel bys pak lepsi kontrolu nad tim, co leze do funkce, plus je to blbuvzdornejsi pro usera.

Reply:
Jo, rozhodně, good point!

Reply:
No, a nejlepší by bylo mít funkci ve tvaru `semtext(... , parametry...)`, kde nemusíš vkládat list, ale rovnou jen názvy těch objektů s modely, ale to neumím udělat.

Reply:
Ted Ti asi nerozumim, jak presne to myslis? Prave proti tomu `funkce(..., parametry)` varuju, protoze mi prijde, ze je jednodussi tam mit jeden argument, co bude list modelu, pro prehlednost a prijemnost kodu.

Reply:
@hynekcigler Hynku, tady Te poprosim jeste o odpoved :) ^